.eslintrc

```diff
@@ -10,10 +10,12 @@
     "es6": true
   },
   "plugins": [
+    "flowtype",
     "prettier"
   ],
   "rules": {
     "consistent-return": "off",
+    "flowtype/define-flow-type",
```

Comment:
The CI test is failing because of this one: https://travis-ci.org/amilajack/eslint-plugin-flowtype-errors/jobs/346625539#L502

Reply:
Whoops, must have hit backspace without realizing. Fixed now.